python/setup.py

```diff
@@ -15,8 +15,28 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import sys
+
 from setuptools import setup
 
+DEPENDENCIES = ['botocore==1.20.14',
+                'boto3==1.17.14',
+                'fastavro==1.3.2',
+                'hmsclient==0.1.1',
+                'mmh3==3.0.0',
+                'pyparsing==2.4.7',
+                'python-dateutil==2.8.1',
+                'pytz==2021.1',
+                'requests==2.25.1',
+                'retrying==1.3.3',
+                'pyarrow==2.0.0'
+                ]
+
+# pandas doesn't get built for 3.6 anymore, we should consider deprecating 3.6
```

Comment:
The whole PyData ecosystem around `pandas`, `numpy` and also soon `pyarrow` drops / has dropped 3.6, so dropping it here would also make sense.